src/lib/serialize-plugins.ts

```diff
@@ -0,0 +1,6 @@
+export function serializePlugins( plugins: string[] ): string {
+	const serializedArray = plugins
+		.map( ( plugin, index ) => `i:${ index };s:${ plugin.length }:"${ plugin }";` )
+		.join( '' );
+	return `a:${ plugins.length }:{${ serializedArray }}`;
+}
```

Comment:
I've created this helper function because serializing this array is straightforward, but if it gets more complex in the future, we could consider using `php-serialize`.

Reply:
I don't expect to use it externally. At first I implemented it as a protected method, but then I thought I'll add tests and maybe it made more sense as and independent function.